src/blueapi/client/event_bus.py

```diff
@@ -28,11 +30,11 @@ def __exit__(self, exc_type, exc_value, exc_traceback) -> None:
 
     def subscribe_to_all_events(
         self,
-        on_event: Callable[[MessageContext, AnyEvent], None],
+        on_event: Callable[[AnyEvent, MessageContext], None],
     ) -> None:
         try:
             self.app.subscribe(
-                self.app.destinations.topic("public.worker.event"),
+                MessageQueue(name="public.worker.event"),
```

Comment:
```suggestion
                MessageTopic(name="public.worker.event"),
```

These were Topics (multiple consumers) before, not Queues (single consumers)